### ISSUE TITLE
Missing information

### ISSUE THREAD

**Issue by Zak-Ghani** [dotnet/AspNetCore.Docs#23435]
[EDIT by guardrex to fix the code blocks. @Zak-Ghani ... Use triple-backticks above and below code blocks to get them to render in GH issue comments. Use single backticks for inline code.]

Overall the doc does not read correctly if like me you are a newbie. e.g. under Add a controller section you say :
Add a BooksController class. but fail to say it should actually be BooksController.cs

Going through and making a lot of assumptions you need to include an additional line to say the following must also be added to step 4 under "Add a CRUD operations service" section

```csharp
using Microsoft.Extensions.Options;
```

as this is a requirement for the step 3 line below

```csharp
services.AddSingleton<IBookstoreDatabaseSettings>(sp =>
    sp.GetRequiredService<IOptions<BookstoreDatabaseSettings>>().Value);
```

`IOptions` ... will be in an error state without it

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5240866d-d0a1-f0dd-7b2c-203e1d69bc4b
* Version Independent ID: 83a5cb3b-23bc-4f0c-cd1c-1d1ffeb1131a
* Content: [Create a web API with ASP.NET Core and MongoDB](https://docs.microsoft.com/en-gb/aspnet/core/tutorials/first-mongo-app?view=aspnetcore-5.0&tabs=visual-studio-code#prerequisites)
* Content Source: [aspnetcore/tutorials/first-mongo-app.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-mongo-app.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @prkhandelwal
* Microsoft Alias: **scaddie**

**Comment by Rick-Anderson** [comment]
Moved to Container list: Create a web API and MongoDB #16196

**Comment by serpent5** [comment]
Fixed in #24126.
